quarkus/config-api/src/main/java/org/keycloak/config/HttpAccessLogOptions.java

```diff
@@ -20,4 +22,22 @@ public class HttpAccessLogOptions {
             .category(OptionCategory.HTTP_ACCESS_LOG)
             .description("A regular expression that can be used to exclude some paths from logging. For instance, '/realms/my-realm/.*' will exclude all subsequent endpoints for realm 'my-realm' from the log.")
             .build();
+
+    // check the CookieType
+    public static final List<String> HIDDEN_COOKIE_VALUES = List.of(
+            "AUTH_SESSION_ID",
+            "KC_AUTH_SESSION_HASH",
+            "KEYCLOAK_IDENTITY",
+            "KEYCLOAK_SESSION",
+            "AUTH_SESSION_ID_LEGACY",
+            "KEYCLOAK_IDENTITY_LEGACY",
+            "KEYCLOAK_SESSION_LEGACY"
+    );
+
+    public static final Option<List<String>> HTTP_ACCESS_LOG_MASKED_COOKIES = OptionBuilder.listOptionBuilder("http-access-log-masked-cookies", String.class)
+            .category(OptionCategory.HTTP_ACCESS_LOG)
+            .description("Set of HTTP Cookie headers whose values must be masked when the 'long' pattern or '%{ALL_REQUEST_HEADERS}' format is enabled with the 'http-access-log-pattern' option.")
+            .hidden() // hidden for now as we do not have the full Quarkus support for this yet
```

Comment:
Does it matter if there's quarkus support whether this is hidden?

Will this ever be set by a user? If so, we could consider making it additive to the default.